### PR TITLE
Fix broken links and remove no longer maintained repositories

### PR DESCRIPTION
**Pull request by Hell01World**

### CODE REVIEW

**Comment by Hell01World** [comment]
While we are at it, we might as well "mark" [Mull as a fork of Fennec](https://gitlab.com/divested-mobile/mull-fenix#whats-this)

File: README.md [hunk 1]
```diff
@@ -447,7 +446,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
     * [Headphone Detector](https://vanilla-music.github.io/) <sup>**[[F-Droid](https://f-droid.org/app/ch.blinkenlights.android.vanillaplug)]**</sup>
 * [**Simple Music Player**](https://github.com/SimpleMobileTools/Simple-Music-Player) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.musicplayer)]**</sup>
 * [**Dialog Music Player**](https://github.com/VishnuSanal/DialogMusicPlayer) <sup>**[[F-Droid](https://f-droid.org/app/phone.vishnu.dialogmusicplayer)]**</sup>
-* [**Phonograph**](https://github.com/kabouzeid/Phonograph) <sup>**[[F-Droid](https://f-droid.org/app/com.kabouzeid.gramophone)]**</sup>
+* [**Phonograph**](https://github.com/kabouzeid/Phonograph)
```
Comment (offa) [inline README.md:449]: I guess we can remove the entry as it's no longer maintained.

File: README.md [hunk 2]
```diff
@@ -566,7 +565,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • System
 
 * [**Dark theme**](https://github.com/phstudio2/Darktheme) <sup>**[[F-Droid](https://f-droid.org/app/com.phstudio.darktheme)]**</sup>
-* [**Privacy Friendly Net Monitor**](https://github.com/SecUSo/privacy-friendly-netmonitor) <sup>**[[F-Droid](https://f-droid.org/app/org.secuso.privacyfriendlynetmonitor)]**</sup>
+* [**Privacy Friendly Net Monitor**](https://github.com/SecUSo/privacy-friendly-netmonitor)
```
Comment (offa) [inline README.md:568]: Can be removed, inactive.

File: README.md [hunk 3]
```diff
@@ -657,7 +656,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Weather
 
 * [**Geometric Weather**](https://github.com/WangDaYeeeeee/GeometricWeather) <sup>**[[F-Droid](https://f-droid.org/app/wangdaye.com.geometricweather)]**</sup>
-* [**Privacy Friendly Weather**](https://github.com/SecUSo/privacy-friendly-weather) <sup>**[[F-Droid](https://f-droid.org/app/org.secuso.privacyfriendlyweather)]**</sup>
+* [**Privacy Friendly Weather**](https://github.com/SecUSo/privacy-friendly-weather)
```
Comment (offa) [inline README.md:659]: Can be removed, inactive.

**Review by offa — APPROVED** [review]
Thanks! :-)

We can remove some of those entries (see comments) due to inactivity, but that's not necessary part of this PR and wont block it.

**Pull request merged**